src/main/resources/assets/computercraft/lua/rom/apis/rednet

```diff
@@ -8,8 +8,16 @@ local tHostnames = {}
 
 function open( sModem )
 	if type( sModem ) ~= "string" then
-		error( "expected string", 2 )
+	    for key,value in ipairs({"left","right","up","bottom","back","front"}) do
```

Comment:
Try to add an if statement here that ensures that there is only one modem attached (when auto selecting modem) so people don't get problems with multiple modems turning on and sending duplicate messages, etc.